lib/hutch/version.rb

```diff
@@ -1,3 +1,3 @@
 module Hutch
-  VERSION = '1.4.0.pre'.freeze
+  VERSION = '1.4.1.pre'.freeze
```

Comment:
Please roll this change back. This PR has nothing to do with version changes and it's up to the maintainer to decide when to change the version and why.